src/server/WebDav.ts

```diff
@@ -327,7 +327,7 @@ export default class WebDav {
 				walker.on('file', (root, fileStat, next) => {
 					const file = resolve(root, fileStat.name);
 					const toFile = relative(isCartridge ?
-						pathToCartridgesDir.replace(processingFolder, '') :
+						pathToCartridgesDir.replace(new RegExp(processingFolder+'$'), '') :
```

Comment:
This is a fix for paths that contain the same name than a cartridge, i.e. `/xyz/some/path/cartridges/xzy`